### ISSUE TITLE
Accessibility with Taiko

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#1689]
**Is your feature request related to a problem? Please describe.**
Elements are identified based on what is visible on the screen. But some elements do not have associated text visible on screen. To test accessibility or perform action on elements without text, we may have to use selectors that are CSS and XPath selectors dependent. As powerful as they can be, they have drawbacks like HTML structure dependency. 

**Describe the solution you'd like**
As mentioned in [Introduction to ARIA](https://developers.google.com/web/fundamentals/accessibility/semantics-aria) ARIA can modify existing element semantics or add semantics to elements where no native semantics exist. Button with no text but an icon is common. The accessibility label helps to understand the functionality associated with it.

```
<button aria-label="Open Filter Menu">
<span class="ag-icon ag-icon-filter" unselectable="on" role="presentation"></span>
</button>
```

To access such elements we should be able to use aria attributes rather than css or xpath.
[CDP has accessibility domain](https://chromedevtools.github.io/devtools-protocol/tot/Accessibility/) . Using the new CDP endpoint we should be able to provide selectors that are accessibility complaint.

**Describe alternatives you've considered**
Currently this can be achieved by `button({"aria-label":"Open Filter Menu"})`. Also on elements which are not explicitly defined we can access it like `$('[aria-colindex="9"]')`. This is not easily discoverable and is not consistent in syntax. 

**Additional context**
Here is how I am currently implementing it
```
const { openBrowser, goto, click, $, within, doubleClick, closeBrowser } = require('taiko');
(async () => {
    try {
        await openBrowser();
        await goto('https://www.ag-grid.com/example.php');
        await click("Columns");
        await click(button({"aria-label": "Open Filter Menu"}),below("Language"));
        await doubleClick($('[col-id="rating"]'), within($('[row-index="2"]')));
    } catch (error) {
        console.error(error);
    } finally {
        await closeBrowser();
    }
})();
```
Please consider adding explicit support for [aria related Semantics](https://developers.google.com/web/fundamentals/accessibility/semantics-aria).
